src/lib/yang/value.lua

```diff
@@ -98,7 +98,14 @@ function types.identityref.tostring(val)
 end
 
 types['instance-identifier'] = unimplemented('instance-identifier')
-types.leafref = unimplemented('leafref')
+
+types.leafref = {}
+function types.leafref.parse(str, what)
+   return assert(str, 'missing value for '..what)
```

Comment:
In some ideal world, the thing we should return as the leafref's parser is not a string parser, but rather a parser for the referred-to node, which might be e.g. an ipv6 address.  i think we can leave that to some future patch tho